sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOptions.cs

```diff
@@ -38,10 +38,17 @@ public RecognizeContentOptions()
         public string Language { get; set; }
 
         /// <summary>
+        /// <para>
         /// Custom page numbers for multi-page documents(PDF/TIFF). Input the number of the
```

Comment:
I think we should update this docstring to remove the part about OCR result.

Reply:
updated